### DISCUSSION TITLE
Include vision in dashboard documentation

### DISCUSSION THREAD

**Discussion by joelostblom** [quarto-dev/quarto-cli#7343] [Q&A]
### Description

I recently found out about Quarto dashboards and it looks really interesting! I wonder if you could add some clarity around the vision of this feature: Is it meant as a full enterprise dashboarding solution similar to shiny, or something quick and simple when a full shiny dashboard is not required? I am aware that shiny can be integrated into quarto dashboard and that probably blurs the line in some cases, but I would still be really interested in hearing more how this Quarto dashboards are envisioned to be used.

**Comment by mcanouil** [comment]
Note that this feature landed in the 1.4 pre-release two days ago.

I don't believe there is a "vision" or target use case as there is no target for format/project such as book or website.

The goal is to be able to build dashboards which are basically just flex grids.
Then there is no real limitations of what you put in it, interactive code using plotly, observable.js, shiny or static content, etc.

**Reply by joelostblom** [reply]
Thanks for the reply @mcanouil . It sounds like quarto dashboards will be quite flexible and I know its still a very new project. I think I could rephrase my question to "When do I use just quarto dashboards on its own, when do I use just Shiny on its own, and when do I combine both together?". I think I can only answer the first one, it seems like quarto dashboards on its own is simpler/easier than shiny, but I am not sure for the other two. Maybe this will become clearer when I start exploring more and trying things out.

**Comment by cscheid** [comment]
We're working on documentation and a large number of examples. Keep an eye on https://quarto.org/docs/dashboards/!

**Reply by joelostblom** [reply]
That's great, thank you!
